plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiMetadataProcessor.kt

```diff
@@ -18,6 +18,8 @@ import org.jetbrains.org.objectweb.asm.Opcodes
 fun abiMetadataProcessor(
     annotationVisitor: AnnotationVisitor,
     removeDataClassCopyIfConstructorIsPrivate: Boolean,
+    classesToBeDeleted: Set<String>,
+    pruneClass: Boolean,
```

Comment:
The idea behind this flag is to get rid of anything in the class that was about to be deleted but left in place because of nested private interface or annotation.